sass/themes/cr/2019/components/just-in-time-block/_just-in-time-block.scss

```diff
@@ -5,4 +5,11 @@
     font-family:$montserrat;
     font-weight: 800;
   }
+  .just-in-time--content {
+    strong {
+      font-weight: inherit;
+      font-family: inherit;
+      text-decoration: underline;

```

Comment:
As this is using the chubby font already, we've got nowhere 'bolder' to go, so am just using the underline for emphasis for now